src/components/OasisMarketWidget.js

```diff
@@ -1,16 +1,82 @@
-import React, { PureComponent } from 'react'
+import React, { PureComponent } from 'react';
 import { PropTypes } from 'prop-types';
+import { last } from 'lodash';
+
 import OasisWidgetFrame from '../containers/OasisWidgetFrame';
+import { OasisTable } from './OasisTable';
+import { ETH_UNIT_ETHER } from '../constants';
+import { TIME_SPAN_DAY, TIME_SPAN_MONTH, TIME_SPAN_WEEK } from '../utils/timeSpan';
+import getTokenPairPrice from '../utils/tokens/getTokenPairPrice';
+import getTokenPairVolume from '../utils/tokens/getTokenPairVolume';
+import getTokenPairTrades from '../utils/tokens/getTokenPairTrades';
+import getTokenPairFormat from '../utils/tokens/getTokenPairFormat';
```

Comment:
There is huge duplication in the names. This is the first thing and the second is that the methods become long and in the code is hard to read. 

What do you think about the following.

``` javascript
 import { trades , format , price , volume } from '../util/tokens/pair '
```

Reply:
:+1:

Reply:
done